pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/replication.tsx

```diff
@@ -7,10 +7,32 @@ import { Metric, Axis, AxisUnits } from "src/views/shared/components/metricQuery
 import { GraphDashboardProps, nodeDisplayName, storeIDsForNode } from "./dashboardUtils";
 
 export default function (props: GraphDashboardProps) {
-  const { nodeIDs, nodesSummary, storeSources } = props;
+  const { nodeIDs, nodesSummary, storeSources, tooltipSelection } = props;
 
   return [
-    <LineGraph title="Ranges" sources={storeSources}>
+    <LineGraph
+      title="Ranges"
+      sources={storeSources}
+      tooltip={(
+        <div>
+          {`Count of ranges ${tooltipSelection} with the following properties:`}
+          <dl>
+            <dt>Ranges</dt>
+            <dd>All ranges</dd>
+            <dt>Leaders</dt>
+            <dd>Raft leader ranges</dd>
```

Comment:
The awkward thing here is that replicas are raft leaders, not ranges.  But since there is expected to be one replica chosen as raft leader per range, it is meaningful to compare the count of replicas with the count of ranges.

All that's to say that I'm not that satisfied with the phrase "Raft leader ranges" but I'm not sure I have a better formulation.

Reply:
"Replicas elected as leader" or perhaps more explicitly "Ranges with leaders elected"